doc/nrf/releases/release-notes-1.7.0.rst

```diff
@@ -0,0 +1,449 @@
+    The report is handled by the nRF Desktop peripherals and forwarded by the nRF Desktop dongles.
+  * Added support for nRF5340 DK working as an nRF Desktop dongle.
+  * Added a functionality for forwarding HID boot reports in :ref:`nrf_desktop_hid_forward`.
+  * Added GPIO LEDs to the ``nrf52820dongle_nrf52820`` board.
+
+* Updated:
+
+  * Changed settings backend from FCB to NVS.
+  * Switched to using :ref:`caf_power_manager`.
+  * Fixed an issue with generating motion in :ref:`nrf_desktop_motion` (``motion_buttons`` and ``motion_simulated``) while the HID boot protocol was in use.
+
+Bluetooth® LE
+-------------
+
+* Added:
+
+  * Production support for :ref:`nRF21540 GPIO <ug_radio_fem_nrf21540_gpio>` for both nRF52 and nRF53 Series.
+  * :ref:`rscs_readme` - This module implements the Running Speed and Cadence Service (RSCS) with the corresponding set of characteristics.
+  * :ref:`peripheral_rscs` sample - This sample demonstrates how to use the Running Speed and Cadence Service (RSCS).
+  * Experimental implementation of the UART async adapter extension inside the :ref:`peripheral_uart` sample.
+
+* Updated:
+
+  * :ref:`ble_samples` - Changed the Bluetooth® sample Central DFU SMP name to :ref:`Central SMP Client <bluetooth_central_dfu_smp>`.
+  * :ref:`direction_finding_connectionless_rx` and :ref:`direction_finding_connectionless_tx` samples - Added default configuration for ``nrf52833dk_nrf52820`` and ``nrf5340dk_nrf5340_cpuapp``, and ``nrf5340dk_nrf5340_cpuapp_ns`` boards.
+  * :ref:`direct_test_mode` - Added an automatic build of the :ref:`nrf5340_empty_app_core` sample, when building for ``nrf5340dk_nrf5340_cpunet``.
+  * Fixed the NCSDK-9820 known issue in the :ref:`peripheral_lbs` sample.
+    When **Button 1** was pressed and released while holding one of the other buttons, the notification for release was the same as for press.
+  * Fixed an issue in the :ref:`gatt_dm_readme` library where a memory fault could happen if a peer device disconnected during the service discovery process.
+  * :ref:`lbs_readme` library - Added write request data validation in the LED characteristic.
+
+Bluetooth mesh
+---------------
+
+* Added:
+
+  * The choice of default Bluetooth® LE Controller changed from Zephyr Bluetooth® LE Controller to SoftDevice Controller.
+  * Bluetooth® mesh profiles and models are ready for production.
+
+* Updated:
+
+  * Updated the :ref:`bt_mesh_light_hsl_srv_readme` and the :ref:`bt_mesh_light_xyl_srv_readme` models to no longer extend the :ref:`bt_mesh_lightness_srv_readme` model, and instead get a pointer to this model in the initialization macro.
+  * Updated samples with support for the :ref:`zephyr:thingy53_nrf5340`.
+  * Fixed an issue where beacons were stopped being sent after node reset.
+  * Fixed an issue where the IV update procedure could be started immediately after the device has been provisioned.
+  * Fixed multiple issues in the :ref:`bt_mesh_sensor_types_readme` module.
```

Comment:
Need to add migration details for mesh

Reply:
Please add as a comment here or a PR to this PR.

Reply:
Waiting for this PR to get merged first: https://github.com/nrfconnect/sdk-nrf/pull/5156

Reply:
Migration now added.